src/client/renderer/r_mesh_model.c

```diff
@@ -342,13 +343,13 @@ static void R_LoadMd3VertexArrays(r_model_t *mod) {
 
 		for (uint16_t i = 0; i < md3->num_meshes; i++, mesh++) { // iterate the meshes
 
-			const r_md3_vertex_t *v = mesh->verts + f * mesh->num_verts;
+			const r_md3_vertex_t *vert = mesh->verts + f * mesh->num_verts;
 			const d_md3_texcoord_t *t = mesh->coords;
 
-			for (uint16_t j = 0; j < mesh->num_verts; j++, v++, ++t) {
-				VectorAdd(frame->translate, v->point, vertexes[j + vert_offset].vertex);
-				VectorCopy(v->normal, vertexes[j + vert_offset].normal);
-				Vector4Copy(v->tangent, vertexes[j + vert_offset].tangent);
+			for (uint16_t j = 0; j < mesh->num_verts; j++, vert++, ++t) {
+				VectorAdd(frame->translate, vert->point, vertexes[j + vert_offset].vertex);
+				VectorCopy(vert->normal, vertexes[j + vert_offset].normal);
+				Vector4Copy(vert->tangent, vertexes[j + vert_offset].tangent);
```

Comment:
Because "v" was used above, was just to silence shadow warning. Probably not; pre-increment is a thing I got into the habit of doing for "for" loops, like I said on IRC, so it's possible you'll see it crop up every now and then